### ISSUE TITLE
New name for this plugin; brainstorming and choosing

### ISSUE THREAD

**Issue by MPolleke** [qtranslate/qtranslate-xt#601]
Hi, very interested in this effor to revive qTranslate. :)  I saw the discussion on:
https://github.com/qTranslate-Team/qtranslate-x/issues/579
I think it would be good to gather ideas (brainstorm phase) and hopefully pick a good name. That way there can be an effort to rename the project, and maybe rename functions and settings (if desired). Just trying to think along.

Some ideas gathered from the old linked issue:
* openQtranslate (mayaliny)
* qTranslateXXL (probably trademark issue) (mayaliny) 
* qTranslate-CE (Community Edition) (probably trademark issue) (abclution)
* qTransferendum (Translate in Latin) (Ruaphoc)
* qTranslate-NG (as in Next Generation) (probably trademark issue) (Ruaphoc)
* qTranslate-WP (probably trademark issue) (Ruaphoc)
* KUKU translate (Italian "q" sound like "ku") (mayaliny)
* qTraduki (qTranslate is esperanto) (mpolleke)

More ideas are welcome.
My preference so far would be qTransferendum, since latin is not a living language anymore and avoids confusion among live speakers.

If it turns out the old slug at wp.org can be used, all the better. If that is not possible or unwanted, a new name is needed.

**Comment by rinurinu** [comment]
Cool suggestions so far! If I understand it correctly it cant start with qTranslateX because of the wordpress plugin guidelines, right?

**Comment by MPolleke** [comment]
That i correct. It is considered a trademark and cannot be used as name or at the start of the name.

**Comment by herrvigg** [comment]
Well i have to rediscuss with the plugin managers at Wordpress. Creating a new plugin with a different name is the second option, we would also need to repackage a lot of stuff and make it clearly different.

But, in some cases, such as when the author disappears, it seems we can be given the right to update the official plugin. However this doesn't give the right to claim the trademark so it can be a bit touchy.

For now it's not very important, we should focus on the code with new fixes and features. In any case we need considerable changes before we can take any of the two options. So we should start thinking about a roadmap. On the readme i mention what i think are the most important features to come: Gutenberg, translatable slugs, WooCommerce.

**Comment by wisamx** [comment]
What characterize qtranslate is that it uses in-same page-translation.
So i feel the name should have something related.
Like: inline translator, itranslator, itranslate.

**Comment by emyawasap** [comment]
I was thinking of `decipher` but I think it was too deep.

**Comment by dschmidtke** [comment]
HI, I suggest newTranslate, since it sounds very much like qTranslate and it's indeed new. Easy. I use qTranslate-x in some own and client's projects, so I want it to survive, and I am willing to contribute code to this fork.

**Comment by phil4literacy** [comment]
I like the name newTranslate.

**Comment by Peter-2by2** [comment]
Like I said on https://github.com/qTranslate-Team/qtranslate-x/issues/579 it's fairly straightforward to take over an obviously abandoned plugin. Personally, I'd concentrate on getting that sorted for now and worry about the name later. It might well be that the "trademark" never gets questioned. If it ever does, deal with it then.

**Comment by herrvigg** [comment]
Yes the name is a secondary question for now and even the WP repo.
First i would like to get some feedbacks on the new release. Have you people tried QT-XT?

**Comment by Peter-2by2** [comment]
Hi Herrvigg. I've installed it on a test site, replacing the original. It's not a production site, so little more than a few test posts and no coded enhancements, but it worked perfectly. No errors so far!

**Comment by wimstefan** [comment]
Same here. Switched on my development environment to QT-XT and all works so far. But I have to admit that I didn't really test it to the max ...

**Comment by MPolleke** [comment]
I tried it and it seems fine.
I also tried adding support for the WP 4.8+ Text widgets, but everything is quite connected with everything, and it seems to be a bit over my head to contribute code.

**Comment by herrvigg** [comment]
For the future my intention is to move to a new repo abandoning some of the new qTranslate-X legacy.
I've been struggling quite a bit to clean up this project but it's a quite a big mess when it comes to the integration! The new project must be easier to maintain so i feel i have to refactor quite many things and probably discard some legacy stuff.
For the moment i'm still on qTranslate-XT as a transition phase and doing some experiments with Gutenberg but from a certain point i feel this project needs a real new start to be able to live on the long run. So yes the question of the name will come back again soon :P

**Comment by majekw** [comment]
@herrvigg , lets face the fact that you are maintaining qTranslate-xt for over 8 months. Just stick to current name as many people already know continuation of qTranslate under this name.
Give up attempts for taking over qTranslate, mqTranslate or qTranslate-X Wordpress repositories, it's waste of time in my opinion.
As for refactoring and tests - Rome wasn't built in a day, just keep doing your great job, but don't wait for this piece of software to be perfect before official release.
Once again, thank you for your great job and I'm looking forward to see qTranslate-XT directly in Wordpress plugins.

**Comment by herrvigg** [comment]
Ahah i was currently writing this long message at the same time! I'm actually thinking to abandon the name qTranslate-XT.

Indeed the status of this project as "qTranslate-XT" is starting to get problematic (e.g. the last PR for composer). Also it would help to become a new official plugin available at wordpress.org. The current version seem to be working good for many people and i understand it's a relief for many users. I'm fine to continue helping with the transition, it should and can be done gradually.

The problem is that i can't hardly push it as qTranslate-XT, the new rules with WordPress are a bit harder, and that's a good thing. We can't have qTranslate as prefix because it's considered by WP as a trademark. In the best case it would be with a prefix to get around it but that's just a kind of trick.

Strictly speaking, to take over qTranslate we'd need to get the approval of the owners to keep the brand. Problem, as everyone knows, there are not active anymore. As i see it they are two: John Clause for qTranslate-X and Qian Qin for qTranslate who wrote actually most of the code still in use. I suppose qTranslate comes from his name, correct me if i'm wrong! But John Clause doesn't respond and Qian Qin also abandoned the project long ago.

For that reasons and to avoid any speculation it's better we finally abandon qTranslate in the name and rebrand it. The previous authors made a huge contribution to this project but we need to move forward. They will be credited as deserved. The drawback is that we might lose some users by fear of a big change but it's better to move on. If the tool is good, people should find it again.

So let's restart some brainstorming!

**Comment by herrvigg** [comment]
Here a few points of discussion:
- The names newTranslate or openTranslate sound a bit too generic to me. Clearly the code will remain open source as most of the WP plugins so we don't need to repeat that.
- I like the idea that the name should represent what makes this tool unique. As mentioned by @wisamx the core idea is in-same page translation. So the name could have something to do with concept as: fusion, assemblage, aggregate, blend, joint, ... Maybe even something with buffer.
- Translate is a bit long to spell. I love the term "lang" which might be easier to combine. But you can try to substitute if it sounds better.
- We might have a dash, special character or upper case for a better look. But these are just minor variations for "styling". 

So here are a few ideas coming on the fly: `fusedlang, langfused, langfusion, agglang, aggrelang, assemblang, langblend, unilang, langutd, langembed, langbuff, langbuff`.

Also, another way to keep a "soft legacy" to qTranslate would be something like: `qtlang, qtxlang, lanqtxt, langcutie, cutielang, langcutee, cuteelang`.

Other suggestions? Someone has a better inspiration?

**Comment by majekw** [comment]
Ok, I see your point.
For me `qtlang` or even `qtlang-xt` (to have better recognition for current users od qTranslate-XT) is fine.
`cuteelang` is cute :-)
So, that's my votes.

**Comment by alphp** [comment]
langPanels, multiLangPanel, langSideToSide
I'm not very witty today ...

**Comment by phil4literacy** [comment]
I also like qtlang, although I find "openTranslate" is a good name too, 
as there are also commercial plugins.

Philip

**Comment by Ruaphoc** [comment]
the assumptions so far are that everyone uses English. Let's not focus on English words/names. Latin is the foundation of a large number of languages, so why not a Latin name? We could shorten the Latin words to make them more appealing:

Change Language -> mutatione linguarum

so why not:

MutaLingua

Greg

**Comment by wimstefan** [comment]
That's a nice idea!
Staying in that line of thought another word popped up immediately in my mind: "TransLingua"

**Comment by herrvigg** [comment]
I'll give a bit of time before deciding for a new name. 

Meanwhile i have other questions coming up with the brand migration:
- git repo: should i create a new one or rename this one? My first idea was to rename because i imported all the history with the tickets in github and i don't want to do it again (the mass notifications have been bothering a lot of people). The counterpart is that it's going to be a bit confusing for those having their local git pointing to qtranslate-xt but with proper notice and info it should be feasible. Some external hyperlinks will also break but i think it's more important to carry the history. Alternatively we could archive this one with its backlog and refer to the old tickets here. What's the best?
- versioning: should i restart from 0.1 or 1.0 or continue from the current version (probably from a 4.x). Somehow i'd like to restart from zero but this is also related to the first question with regard to the releases and all the history. It might also depend if it is a continuation or a clear break.
- options: from that point the new tool will not share the options anymore with qTranslate-X so i need to find the good way to migrate/import all those. It might be the opportunity to rethink a bit how they are handled, i find the code very hard to understand/maintain for all this part. This is maybe the main point to be solved before being able to restart somewhere else. Apart from this I don't plan to do any DB migration for the translated data itself (in the posts and so on).

**Comment by herrvigg** [comment]
_Edit_: i moved those questions to a new ticket https://github.com/qtranslate/qtranslate-xt/issues/648. Let's only discuss about the name here.

**Comment by Ruaphoc** [comment]
I actually really like the “TransLingua” suggestion!

**Comment by bagaweb** [comment]
TransLingua sounds quite weird in Italian... Something like Transgender's tongue...

**Comment by alphp** [comment]
I am Spanish, and although it is true that someone who was badly thought could distort TransLingua, I think it will be anecdotal.

**Comment by Peter-2by2** [comment]
I recently came across this word - Metaphrase, which means a "literal translation". I think that sounds pretty apropos. If you wanted to tie it back to the original, maybe qMetaphrase?

**Comment by herrvigg** [comment]
I also speak italian and i agree TransLingua sounds weird. I must say i'm not fan of the Lingua term because it refers more to the tongue as organ than language in many... languages 😛

**Comment by alphp** [comment]
In Spanish there is a word ending in lingus that is not suitable for all audiences.

In that sense, I prefer English, it's more neutral.

I like CuteTranslate ( @TheHeat in https://github.com/qtranslate/qtranslate-xt/issues/648#issuecomment-470200007 ) because it sounds good and is reasonably similar/different to qTraslate.

**Comment by Blackbam** [comment]
How about DirectTranslate or InstaTranslate (or something similar)? I think it would describe the Plugin best, as there is no unnecessary duplication of any content. Just the text within a post or fields which belong to a post are translated.

**Comment by herrvigg** [comment]
So far i like Cute Translate most :) CUTL!

**Comment by herrvigg** [comment]
What sounds/look best, Cute Translate or CuTranslate?

**Comment by Blackbam** [comment]
Not sure XD

**Comment by reddo** [comment]
> What sounds/look best, Cute Translate or CuTranslate?

CuteTranslate

**Comment by TheHeat** [comment]
I think the simpler Cute Translate will be easier for people to remember - the success of a plugin often relies on users being able to recommend it to others and often a name that is concatenated or 'too clever' risks completely passing people by. I say this as someone who's previously run a business where every time I told someone the name I had to spell it. It's a drag. 

Better to keep it simple

**Comment by pszurgot** [comment]
CuteTranslate is cool :)

**Comment by dschmidtke** [comment]
CuteTranslate is fine, since it sounds very much like the original name qTranslate.

**Comment by wimstefan** [comment]
If I have to choose between the two CuteTranslate it is! 😏

**Comment by almendro** [comment]
my vote for CuteTranslate

**Comment by damen02** [comment]
Hi everybody. What a good news that this project is alive again!

I agree this may be not a priority by now, but it's still important.
I have some experience on naming things like projects, products, and other stuff (have a background on branding, graphic and web design). 

Here are some ideas that could help.
- One point to consider is de ease of pronunciation. When you read a word your mind "says it" as part of the process of memory retention. For example,  cuteTranslate looks good in concept, but is tricky for speaking.
- The reference to parent project is nice tribute, but don't try to reference too many things because it only works on creators minds.  
- Simplicity is always good, think about future name addons. But don't forget to be unique.
That said, something like **"qLang"** sounds a good option to me. It is simple (like the spirit of this project), brief still unique, reference to original, simply to say, easy to remember, and easy to get a nice logo.

Thanks a lot to contributors and goog luck!

**Comment by herrvigg** [comment]
That's a good point, maybe cuteTranslate is a bit too long. I like the idea of something short so you can also think of extensions derived from it.

**Comment by bmachuel** [comment]
What about ez-lang?

--  
Benoît Machuel  

De: Herr Vigg <notifications@github.com>(mailto:notifications@github.com)
Répondre: qtranslate/qtranslate-xt <reply@reply.github.com>(mailto:reply@reply.github.com)
Date: 26 avril 2020 at 15:33:27
À: qtranslate/qtranslate-xt <qtranslate-xt@noreply.github.com>(mailto:qtranslate-xt@noreply.github.com)
CC: Subscribed <subscribed@noreply.github.com>(mailto:subscribed@noreply.github.com)
Sujet: Re: [qtranslate/qtranslate-xt] New name for this plugin; brainstorming and choosing (#601)

**Comment by alelkin** [comment]
QML - QMultiLang

**Comment by herrvigg** [comment]
Before we can make any migration to a new repo and make the plugin official we should decide for a good name so let's try to make some progress here.

@damen02 raised some good points. Indeed maybe we don't need to keep the reference to qTranslate in the name, most of the current users will find the new plugin from here even if it has a different name. If there are enough users downloading it, it doesn't really matter. The legacy for the previous contributions will remain clear but we move now to something new and we should look forward!

I also like the idea that the name should be easy to pronounce but we also want the name to tell something about the nature and/or the spirit of the project.

I'm more in favor of using a `lang` particle rather than `translate` because it's shorter. Also, _translatation_ is not really what this plugin is about, since you have to do the translation by yourself. What the plugin really does is to handle the different languages in single post or fields. It's about managing the content for edition, storage and display. I want the plugin to remain free, open source and community-driven.

We should differentiate the name from the main other "concurrent" ones: WPML (WordPress MultiLingual), Polylang, Weglot. The other plugins are rather about translation.

Here are a few new suggestions coming to my mind:
- `OpenLang` or `OpLang` to focus on the open-source aspect, the second version being more compact
- `Populang` or `PopuLang` which can be seen as a contraction of popular or population with language
- `EasyLang` or `EaseLang` in the same fashion as `ez-lang` or `EzLang` proposed earlier

I tried some combinations to refer to the technical functionality (single post) with words like: _embedded, tag, block, composition, compound_ but it doesn't really work. _Block_ is rather a reserved word now referring to the block editor Gutenberg and we are not handling this on the block level yet.

**Comment by Ruaphoc** [comment]
EasyLang gets my vote, or we could use SimpleLang.

**Comment by wimstefan** [comment]
Tinkering over your wish to have `lang` as main part of the new naming *and* being easy pronounceable  I came up with "LangWhich" as a name ... it's pronounced the same as `language` but also includes the possibility which language to choose.

I hope this is not too silly but I thought I'd mention it anyway ...

**Comment by Blackbam** [comment]
One of the greatest benefits of qTranslateXT is that you do not have to duplicate posts and you also do not have to duplicate the whole page (which is especially awesome with custom eCommerce websites and post types in general that have a lot of meta data which does NOT have to be translated).

Therefore I like _EzLang_ or _EzTrans_ because the simple task of translating a website which is 95% the same content should not overcomplicate things.

**Comment by bagaweb** [comment]
> I'm more in favor of using a lang particle rather than translate because it's shorter. Also, translation is not really what this plugin is about, since you have to do the translation by yourself.

Totally agree with you.

I'd vote for EasyLang or OpenLang.

**Comment by Blackbam** [comment]
Seems like we got a lot of votes for EasyLang?

**Comment by damen02** [comment]
Hi everybody. I'm pretty busy right now but plan to make some comments soon.

**Comment by damen02** [comment]
Hi everybody, I'm sorry to be so late, I'm having a very busy time.

I agree almost everything with HerrVigg,
- The reference to qTranslate may be elsewhere, not necessarily in the name.
- I agree that lang is better than translate.
- I like the idea of the plugin remaining free, but... I’ve seen many good free projects to be abandoned (this included). I think the best model is when there are some optional paid complements that maintain someone involved and compromised with the project. 
For example, translation could be part of a future paid complement.

That said, I like very much **EasyLang** and **EaseLang**, they are 
- compact enough, easy to be said
- spirit referencing,
- very direct, clear, and easy to remember
They have all points in favor.

**Comment by herrvigg** [comment]
It seems EasyLang already exists :-/
https://www.templatemonster.com/landing-page-template/easylang-language-school-one-page-classic-html-landing-page-template-77059.html

Not that it is a pure WordPress plugin but it's web-related and there is some kind of connection to WordPress so better avoid this to avoid confusions and conflicts.

**Comment by bagaweb** [comment]
I'd vote for EaseLang, EzLang or OpenLang then.

**Comment by herrvigg** [comment]
EaseLang: exists on Facebook https://www.facebook.com/easelang
EzLang: other language related software https://www.ezlang.net/en/
OpenLang:  there is an organization called OpenLang Network https://www.openlangnet.eu/

Not easy to be orginal, but in fact we have to check this and try to find something really unique.

**Comment by Blackbam** [comment]
How about `EasyMultiLang`? Seems to be not in use yet.

**Comment by bagaweb** [comment]
What about WPEasyLang?

**Comment by herrvigg** [comment]
I would like to avoid acronyms which look perhaps right but they are harder to pronounce. I'm sure we can find a more "friendly" name and unique at the same time.

Here are a few points suggested by @damen02 which i like and i believe we should restart from this:

> * One point to consider is de ease of pronunciation. When you read a word your mind "says it" as part of the process of memory retention. For example,  cuteTranslate looks good in concept, but is tricky for speaking.
> * The reference to parent project is nice tribute, but don't try to reference too many things because it only works on creators minds.
> * Simplicity is always good, think about future name addons. But don't forget to be unique.
>   That said, something like **"qLang"** sounds a good option to me. It is simple (like the spirit of this project), brief still unique, reference to original, simply to say, easy to remember, and easy to get a nice logo.

Just for the last part, it's enough to keep a `Lang` particle, we don't need the 'q' necessarily. But if we find something better other than `Lang` why not.

As an extra requirement, we really have to check there is no other language/translation software too close to the new name. As we go for a new plugin, let's differentiate it from anything else. We have this topic opened for 2 years so it's not like we are in a hurry... 😅 but let's move on! 😎

**Comment by bagaweb** [comment]
`That said, something like "qLang" sounds a good option to me. It is simple (like the spirit of this project), brief still unique, reference to original, simply to say, easy to remember, and easy to get a nice logo.`

I agree with your points and I like the name qLang.

**Comment by TheHeat** [comment]
+1 for qLang it's nice, short and simple

**Comment by herrvigg** [comment]
What about `ubiqlang` or `ubiquilang` ? The root is from the latin _ubiquitous_ which means _everywhere_, and it somehow carries the `q` from the qTranslate legacy. It could also be written `ubiqLang` or `UbiqLang` alternatively `UbiquiLang`.

**Comment by bagaweb** [comment]
I'd prefer ubiqLang rather than ubiquiLang, shorter and simpler.

**Comment by damen02** [comment]
From this set I like 'UbiqLang'. The main improve here is it's uniqueness. Downsides? perhaps, a bit hard to understand o pronounce.

**Comment by wimstefan** [comment]
Having to choose between the two I'd also vote for 'UbiqLang' being shorter and more fluent to pronounce.

When I read this new proposition my associations started to work again and being a pasta lover the word "Languini" came into my mind. The familiarity with "linguine" is intended since that means "little tongues". I know you guys are not very keen of associations with "lingua" as in "tongue" but I like the softness of the pronunciation of that word ... ;-)

**Comment by herrvigg** [comment]
> From this set I like 'UbiqLang'. The main improve here is it's uniqueness. Downsides? perhaps, a bit hard to understand or pronounce.

That summarizes it well. Indeed it's a bit difficult to pronounce.

**Comment by herrvigg** [comment]
Recently I was thinking about `qTranslang` or `qTranxlang`. But what about `Intralang`, alternatively `IntraLang`?

The concept of _intra_ goes really well with the technical feature of qTranslate which fuses several languages _within_ a single content (post or field).

**Comment by LaMpiR** [comment]
Do we have maybe any progress on this? They all sound quite good to be honest. WP Openlang is maybe one option as well, since you are here and you are quite open about it.

**Comment by Komarovski** [comment]
> Recently I was thinking about `qTranslang` or `qTranxlang`. But what about `Intralang`, alternatively `IntraLang`?
> 
> The concept of _intra_ goes really well with the technical feature of qTranslate which fuses several languages _within_ a single content (post or field).

IntraLang - is cool! :)

**Comment by Blackbam** [comment]
Also like Intralang :+1:

**Comment by TheHeat** [comment]
Of the last batch of suggestions I'm most drawn to qLang 

It's the most simple to say and/or recall and nods to the project's history

**Comment by herrvigg** [comment]
I'll come back to this.
Ideally we want something that
1. is about the plugin, but not necessarily about the "q" history
2. easy to pronounce and remember
3. unique, at least in WordPress but also try to avoid conflicts with other language/translation software

It's harder than it seems 😅

**Comment by Gregory-N-able** [comment]
"Comm Badge". A nod to the Universal Translator built into the Star Trek Comm Badges.

Another:

"Translation Matrix", a nod to Doctor Who and the Translation Matrix of the TARDIS which translated almost all languages in real-time.

**Comment by spleen1981** [comment]
> Recently I was thinking about `qTranslang` or `qTranxlang`. But what about `Intralang`, alternatively `IntraLang`?
> 
> The concept of _intra_ goes really well with the technical feature of qTranslate which fuses several languages _within_ a single content (post or field).

Could also be shorter: `InLang` sounds good to me.
This refers to the basic idea that no separate entry in the database is used, as multilingual content can be added "inline".

Longer alternatives can be:
- `NewInLang` suggesting that something older has been used as a starting point
- `OpenInLang` suggesting that the plugin is not commercial and community driven

Both acronyms happen to have a meaning to play with...

Actually leaving behind the word _translate_ seems correct to me, as the plugin actually does not offer any tool to translate anything (out of scope).

**Comment by bmachuel** [comment]
EasIdiom, a combination of EASY (one characteristic of qTranslate-XT) and IDIOM (a synonym for language)



Le 25 février 2022 à 10:13:45, Giovanni Cascione ***@***.***) a écrit:

Recently I was thinking about qTranslang or qTranxlang. But what about Intralang, alternatively IntraLang?

The concept of intra goes really well with the technical feature of qTranslate which fuses several languages within a single content (post or field).

Could also be shorter: InLang sounds good to me.
This refers to the basic idea that no separate entry in the database is used, as multilingual content can be added "inline".

Longer alternatives can be:

NewInLang suggesting that something older has been used as a starting point
OpenInLang suggesting that the plugin is not commercial and community driven
Both acronyms happen to have a meaning to play with...

Actually leaving behind the word translate seems correct to me, as the plugin actually does not offer any tool to translate anything (out of scope).

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by picasso** [comment]
> Could also be shorter: InLang sounds good to me.

like `InLang`

**Comment by jed84** [comment]
Propozycja ode mnie:
- LinguifyUX
